launcher/net/HttpMetaCache.h

```diff
@@ -113,7 +113,7 @@ class HttpMetaCache : public QObject {
 
     // evict selected entry from cache
     auto evictEntry(MetaEntryPtr entry) -> bool;
-    void evictAll();
+    auto evictAll() -> bool;
```

Comment:
```suggestion
    bool evictAll();
```
Code-wise, the auto notation should be used only when returning an object/enum from the same class to minimize the code(omit the class name)
This change now adds more characters than needed(repeat in the cpp file).